pepper-apis/docs/specification/src/components/schemas.yml

```
@@ -1516,8 +1516,17 @@ Validation.IntRange:
 AnnouncementModel:
   type: object
   required:
+    - guid
+    - permanent
     - message
   properties:
+    guid:
+      type: string
```

If we're using a real UUID for this, we can define this using the `format` property. It's not explicitly defined by the OpenAPI spec, but we can use arbitrary strings here (tooling should just fall back to the `type` if the `format` is undefined)
```suggestion
      type: string
      format: uuid
```

---

Oh nice